README.md

```diff
@@ -1 +1,73 @@
-# judgement-day
\ No newline at end of file
+# judgement-day
+***
+## Feel Free to Rate FACN2 Mentors
+***
+### How to use our Website:
+To check out the app, Please clone this Repository.
+```javascript
+git clone  https://github.com/facn2/judgement-day.git
```

Comment:
im not sure this is the right url click on the button to clone the repo and check what is the right url